modules/apigee_edge_teams/src/Entity/Controller/TeamController.php

```diff
@@ -233,4 +235,39 @@ public function delete(string $entity_id): EntityInterface {
     return $entity;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getEntities(PagerInterface $pager = NULL, string $key_provider = 'id'): array {
```

Comment:
Pagination not working after inheriting `getEntities()`.

Reply:
Hi @shishir-intelli  The required changes are made.